shared/SECURITY.md

```diff
@@ -4,17 +4,17 @@
 
```

Comment:
Can we also increment the version from `0.0.6` to `0.0.7`?

Reply:
Added a new commit incrementing the version. Thanks for catching that.